build.sbt

```diff
@@ -578,10 +593,16 @@ def customCommands: Seq[Setting[_]] = Seq(
     "clean" ::
       "checkCredentials" ::
       "conscript-configs" ::
+      "interfaceProj/compile" :: // Java project needs to compile first
```

Comment:
Ok, this seems pretty broken then.  Not really a huge fan of sbt-doge if it does this.